Sections/US-National/IAB Privacy’s National Privacy Technical Specification.md

```diff
@@ -254,12 +258,16 @@
 <tr>
 <td>KnownChildSensitiveDataConsents</td>
 <td>N-Bitfield(2,2)</td>
-<td>Two bits for each Data Activity:<code>0</code>  Not Applicable. The Business does not have actual knowledge that it Processes Personal Data or Sensitive Data of a Consumer who is a known child.<p><code>1</code> No Consent<p><code>2</code> Consent&nbsp;<p>(1) Consent to Process the Consumer&rsquo;s Personal Data or Sensitive Data for Consumers from Age 13 to 16.<p><p>References:
-<ul>
-<li>Cal. Civ. Code Cal. Civ. Code 1798.120(c)</li>
-<li>Conn. PA 22-15, Sec. 6(a)(4)</li>
-</ul>
-(2) Consent to Process the Consumer&rsquo;s Personal Data or Sensitive Data for Consumers Younger Than 13 Years of Age.<p><p>References:
+<td>Two bits for each Data Activity:<code>0</code>  The Business does not have actual knowledge that it Processes Personal Data or Sensitive Data of a Consumer who is under 16 years old.<p><code>1</code> No Consent<p><code>2</code> Consent&nbsp;
+  <p>Data Activities:</p>
+  <p>(1) Consumer Consent to Process the Consumer’s Personal Data or Sensitive Data for selling, sharing, or targeted advertising for a Consumer  between the age of 13 and 16.
+  <p>References:
+    <ul>
+      <li>Cal. Civ. Code Cal. Civ. Code 1798.120(c)</li>
+      <li>Conn. PA 22-15, Sec. 6(a)(4)</li>
+    </ul>
+  <p></p>(2) Verifiable consent obtained from the Consumer’s parent or lawful guardian to Process the Personal Data or Sensitive Data of a Consumer Younger Than 13 Years of Age.
```

Comment:
why is younger capitalized?

Reply:
This was there from the original requirements document.